### PR TITLE
Acts 10

### PR DESCRIPTION
**Pull request by EvanZhouDev**
Acts 10 Tasks:
- [x] Section 1 ("Get up and eat")
  - [x] Assets
  - [x] Animations
- [x] Section 2 (Godly Jailbreak)
  - [x] Assets
  - [x] Animations
- [x] Other touches
  - [x] Undo currentPage number to 0
  - [x] Reviews
  - [x] Final fixes

### CODE REVIEW

**Comment by ThatXliner** [comment]
Evan, you should probably change the name of the section from “the church” to something that describes this better

**Review by DragonXDev — COMMENTED** [review]
Consider using a gradient in acts10_12.html because a gradient is used in other Acts chapters.

File: public/assets/styles/acts10_12.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  width: 25%;
+  right: 5%;
+}
+.words9 {
+  position: absolute;
+  top: 4000px;
+  width: 25%;
+  right: 5%;
+}
+.words10 {
```
Comment (DragonXDev) [inline public/assets/styles/acts10_12.css:61]: Namng conventions (for all Acts chapters)…

**Review by ThatXliner — COMMENTED** [review]
Looks ok, if not slightly basic. I would like the text to fade in and out when they're necessary/relevant to the scene. That way we won't see text out of context.

**Comment by ThatXliner** [comment]
I would also prefer if there were a bit more detail or background. But that applies to all of our animations so far and would be part of the "finishing touches and polishing" done in the coming week.

We want to finish the project preferably a week before the actual deadline since the week of the deadline is very stressful, and a lot of other projects are due then as well. @EvanZhouDev should probably get started on the polishing (e.g., *changing the color scheme*) very soon since he has no extracurriculars.

**Comment by EvanZhouDev** [comment]
I've made the heading text a gradient, as @DragonXDev requested.
I've also made the text fade in and out to make the important stuff stand out as @ThatXliner requested.

I believe that this would count as finished, or maybe even 'polished' at this point.

**Comment by ThatXliner** [comment]
:shipit:

**Pull request merged**